tracker/iou_tracker.py

```diff
@@ -280,13 +287,24 @@ def _associate_instances_to_tracks(
     return list_of_matches, unmatched_instances, unmatched_tracks
 
 
-def read_panoptic_image(path: Text, label_divisor: int) -> np.ndarray:
+def read_panoptic_image_2ch(path: Text, label_divisor: int) -> np.ndarray:
   """Reads in a panoptic image in 2 channel format and returns as np array."""
   with tf.io.gfile.GFile(path, 'rb') as f:
     image = tf.cast(tf.io.decode_image(f.read()), tf.int32).numpy()
+
+  if image.shape[2] == 3 and np.any(image[..., 2] != 0):
+    raise ValueError('The input %s is not in 2 channel format.' % path)
   return image[..., 0] * label_divisor + image[..., 1]
 
 
+def read_panoptic_image_3ch(path: Text, label_divisor: int) -> np.ndarray:
+  """Reads in a panoptic image in 3 channel format and returns as np array."""
```

Comment:
Briefly explain what is panoptic image in 3 channel

Reply:
Done.